BookingsApi/BookingsApi/Validations/V1/AddJudiciaryParticipantsToHearingRequestValidation.cs

```diff
@@ -0,0 +1,41 @@
+using BookingsApi.Contract.V1.Requests;
+using FluentValidation;
+using FluentValidation.Results;
+using ValidationFailure = FluentValidation.Results.ValidationFailure;
+
+namespace BookingsApi.Validations.V1
+{
+    public abstract class AddJudiciaryParticipantsToHearingRequestValidation : AbstractValidator<List<JudiciaryParticipantRequest>>
+    {
+        public const string NoParticipantsErrorMessage = "Please provide at least one participant";
```

Comment:
Could you implement the AbstractValidator using the ctor and RuleFor as intended by the library? The UseCustom Rule would be better suited here

Reply:
thanks, had this originally but it got lost when I had to implement the custom validation in order to render the error responses as desired.

added this rule in, now calling it via

`var result = await ValidateAsync(requests);`

Reply:
the above line doesn't invoke the library's ValidateAsync so the rule in the ctor won't actually execute

something like this should do the trick
```
RuleForEach(list => list).SetValidator(new JudiciaryParticipantRequestValidation());
```